packages/default/scss/grid/_layout.scss

```diff
@@ -90,7 +90,7 @@
                 outline: none;
             }
 
-            .k-placeholder-line {
+            .k-skeleton {
                 width: $kendo-grid-cell-placeholder-line-width;
             }
```

Comment:
Do we really need to enforce 80% width for the Grid ? We can verify this with the design team and act accordingly.